pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -1789,11 +1800,25 @@ public void RestoreCoverViewOnScreen(bool hasSavedView = true)
             if (hasSavedView)
                 RestoreViewNameOnScreen("cover");
             // Make sure Skyline is the standard size for a cover shot - Window size and screen shot size differ
-            // And center it in the screen to have the best chance of not needing to move it before Alt-PtrSc
+            SetSkylineWindowSize(1200, 800);
+        }
+
+        // Make the Skyline window as large as possible, without actually putting it into
+        // Maximized state which prevents further resizing
+        const int marginW = 14;
+        const int marginH = 7;
+        public void MaximizeSkylineWindow()
+        {
+            SetSkylineWindowSize(int.MaxValue - marginW, int.MaxValue - marginH);
```

Comment:
int.MaxValue is more than 2 billion.
does it really make a difference if you subtract marginW?

Reply:
I'll add a comment "// So we don't roll into negative territory when SetSkylineWindowSize adds margins"